cmd/cluster/context.go

```diff
@@ -276,6 +280,9 @@ func (o *contextOptions) printLongOutput(data *contextData, w io.Writer) {
 
 	// Print User Banned Details
 	printUserBannedStatus(data, w)
+
+	// Print Migration Status
+	printMigrationStatus(data, w)
```

Comment:
Let's make this more explicit, you could imagine other migrations in the future.
```suggestion
	// Print SDNtoOVN Migration Status
	printSDNtoOVNMigrationStatus(data, w)
```

Reply:
Can confirm that OCM team has more migration types in their future plans